releasenotes/notes/0.6/fix-expval-mit-c2c2701513129316.yaml

```diff
@@ -0,0 +1,11 @@
+---
+fixes:
+  - |
+    Fixed an issue in the expectation value method,
+    :meth:`~qiskit.ignis.mitigation.CompleteExpvalMeasMitigator.expectation_value`, of the error mitigation classes (ie
+    :class:`~qiskit.ignis.mitigation.CompleteExpvalMeasMitigator`)
```

Comment:
Actually the complete mitigator class was the only one that was fine. This fixed applied to the tensored and CTMP mitigators

Reply:
ok, I'll update this and call out both classes